database/database.go

```diff
@@ -2,12 +2,13 @@ package database
 
 import (
 	"database/sql"
-	"errors"
 	"fmt"
 	"time"
 
-	_ "github.com/lib/pq"
 	"gopkg.in/src-d/framework.v0/configurable"
+	errors "gopkg.in/src-d/go-errors.v1"
```

Comment:
`go-errors.v1` package name is already `errors`.

Reply:
goreturns (or goimports, idk, one of those) automatically does this. It might be to facilitate the pkgname -> import path mapping while parsing and doing its thing.